packages/mds-ingest-service/tests/index.spec.ts

```diff
@@ -191,6 +191,23 @@ describe('Ingest Service Tests', () => {
     await IngestRepository.deleteAll()
   })
 
+  describe('getDevices', () => {
+    beforeEach(async () => {
+      const name = await IngestServiceClient.name()
+      await IngestRepository.createDevices([TEST_TNC_A, TEST_TNC_B])
+    })
+    describe('all_devices', () => {
```

Comment:
It's a nit, but what about adding a test to call `getDevices` with both `DEVICE_UUID_A` and an invalid UUID to ensure it throws an exception? That way we'll know if someday the implementation changes to return a partial result instead. Just a thought.

Reply:
It's a good nit. Added two new tests for valid and invalid uuids.